src/Forms/Input/Editor.php

```diff
@@ -38,12 +38,10 @@ class Editor extends Input
     /**
      * Create a tinyMCE rich-text editor input.
      * @param  string  $name
-     * @param  string  $guid
      */
-    public function __construct($name, $guid)
+    public function __construct(string $name)
```

Comment:
If the constructor is changing we will need to update the [FormFactory ](https://github.com/GibbonEdu/core/blob/v25.0.00/src/Forms/FormFactory.php#L167)as well. Ideally, in the future, the planner scripts will be refactored to use the editor through the Form library rather than instantiating it on its own (those are some of the final areas in the core needing forms refactored).

Reply:
Sure :-)